### PR TITLE
72-BUNSPACE2024-Allow-Profile-To-Be-Connected-UI

### LINKED ISSUE #72: Allow Profile to be connected to ui

### PR DESCRIPTION
**Pull request by jmorales8**
add basic connection from backend to ui

## Extra Resources

- [The 10 commandments of navigating code reviews](https://techbeacon.com/app-dev-testing/10-commandments-navigating-code-reviews)
- [Google's Code Review Developer Guide](https://google.github.io/eng-practices/review/)

Closes #72

### CODE REVIEW

File: ui/src/App.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ import { LoginButton } from "./components/LoginButton/LoginButton";
 function App() {
   return (
     <div className="app">
+
```
Comment (GrpFruit) [inline ui/src/App.tsx:20]: remove this line

File: ui/src/pages/Login/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export function Login() {
+  return(
+    <button>This is my button</button>
+  )
+}
```
Comment (GrpFruit) [inline ui/src/pages/Login/Login.tsx:7]: remove login stuff

File: ui/src/pages/Lore/Lore.tsx [hunk 3]
```diff
@@ -1,20 +1,58 @@
...
+        const response = await fetch("/lore");
+        const result: ApiResponse = await response.json();
+        setLore(result.lore);
+        setSectionedBackground(result.lore[currentLoreSelection].background.split("/n"))
+      } catch (error) {
+        console.error("Error fetching data:", error);
+      } finally {
+        setLoading(false);
+      }
+    };
```
Comment (GrpFruit) [inline ui/src/pages/Lore/Lore.tsx:31]: this should be a separate ticket or update this one to include it

**Comment by jmorales8** [comment]
@GrpFruit done

**Review by GrpFruit — APPROVED** [review]

**Comment by GrpFruit** [comment]
this is approved?!?! @jmorales8

**Comment by jmorales8** [comment]
> this is approved?!?! @jmorales

I think, but if i remember there was a bug or someshit with login so i have to fix it before it fix it

**Comment by GrpFruit** [comment]
> bug or someshit with login

Which Bruh commit has the fix? what was the bug as far as I remember I think it looked good

**Comment by jmorales8** [comment]
> > bug or someshit with login
> 
> Which Bruh commit has the fix? what was the bug as far as I remember I think it looked good

Basically, its just an issue with rebasing it did work, however, we want both changes and github just makes mine the one we want automatically so i have to manually fix it

**Comment by GrpFruit** [comment]
Oh I see, thanks for that. Well, it looks good. Add an empty line at the end of the file

**Comment by jmorales8** [comment]
> Oh I see, thanks for that. Well, it looks good. Add an empty line at the end of the file

i just literally fixed it rn, check again

**Comment by jmorales8** [comment]
> > Oh I see, thanks for that. Well, it looks good. Add an empty line at the end of the file
> 
> i just literally fixed it rn, check again

good to merge in @GrpFruit ?

**Comment by GrpFruit** [comment]
> > > Oh I see, thanks for that. Well, it looks good. Add an empty line at the end of the file
> > 
> > 
> > i just literally fixed it rn, check again
> 
> good to merge in @GrpFruit ?

merge

**Pull request merged**